Sources/SwiftInspectorVisitors/Tests/TypeDescriptionSpec.swift

```diff
@@ -138,107 +224,49 @@ final class TypeDescriptionSpec: QuickSpec {
       }
     }
 
-    describe("encode(to encoder: Encoder)") {
+    describe("When decoding a TypeDescription data created with the current library version") {
```

Comment:
Currently the sentences read like:
> When decoding a TypeDescription data created with the current library version
> > utilizing a simple type
>>> successfully encodes the data

Does this more clearly communicate what we're testing?

> With TypeDescription data created with the current library version
> > for a simple type
>>> successfully decodes the data

Reply:
Good point: #73